adk/mlops.py

```diff
@@ -0,0 +1,44 @@
+import yaml
+import json
+import os
+import subprocess
+
+
+class MLOps(object):
+    spool_dir = "/tmp/ta"
+    agent_dir = "/opt/mlops-agent/datarobot_mlops_package-8.1.2"
+
+    def __init__(self, api_token, path):
+        self.token = api_token
+        if os.path.exists(path):
+            with open(path) as f:
+                mlops_config = json.load(f)
+        else:
+            raise Exception("'mlops.json' file does not exist, but mlops was requested.")
+        if not os.path.exists(agent_dir):
+            raise Exception("environment is not configured for mlops.\nPlease select a valid mlops enabled environment.")
+        self.endpoint = mlops_config['datarobot_api_endpoint']
```

Comment:
The env var used by the monitoring agent is called `MLOPS_SERVICE_URL`; not sure if you want to align this naming with that as well, for consistency. Throughout the DR codebase there seem to be a fair number of different strings used for this, but seems like might be worthwhile to match up with the monitoring agent.

Reply:
added datarobot_ to the beginning of that prefix; because these are used external to the datarobot product I think having a distinct naming variation here is useful; they could be confused with an algorithmia specific endpoint